nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ParseCEF.java

```diff
@@ -270,9 +265,8 @@ public void process(final InputStream in) throws IOException {
             // parcefoneLocale defaults to en_US, so this should not fail. But we force failure in case the custom
             // validator failed to identify an invalid Locale
             final Locale parcefoneLocale = Locale.forLanguageTag(context.getProperty(DATETIME_REPRESENTATION).getValue());
-            final boolean validateData = context.getProperty(VALIDATE_DATA).asBoolean();
             final boolean acceptEmptyExtensions = context.getProperty(ACCEPT_EMPTY_EXTENSIONS).asBoolean();
-            event = parser.parse(buffer, validateData, acceptEmptyExtensions, parcefoneLocale);
+            event = parser.parse(buffer, false, acceptEmptyExtensions, parcefoneLocale);
```

Comment:
I'm not familiar with CEF, so take this with a grain of salt. 
Do we no longer support validation in the processor? If so, the corresponding property should be deprecated / removed most likely?
As far as I could see, it is not used anymore other than declaring it a supported property.

Reply:
Thanks for highlighting this change @EndzeitBegins. The `ParseCEF` Processor relies on the ParCEFone library, which currently depends on javax.validation to due data validation. Based on the other Jakarta upgrades, this is not compatible, but disabling data validation allows us to continue using the library for now. For the purpose of this PR, I will remove the Validation property from this Processor, and it is something that could be revisited.